chainer/functions/connection/n_step_rnn.py

```diff
@@ -0,0 +1,882 @@
+            Only ``ws[0][j]`` where ``0 <= j < 1`` is ``(I, N)`` shape as they
+            are multiplied with input variables. All other matrices has
+            ``(N, N)`` shape.
+        bs (list of list of chainer.Variable): Bias vectors. ``bs[i]``
+            represnents biases for i-th layer.
+            Each ``bs[i]`` is a list containing two vectors.
+            ``bs[i][j]`` is corresponding with ``b_j`` in the equation.
+            Shape of each matrix is ``(N,)`` where ``N`` is dimention of
+            hidden units.
+        xs (list of chainer.Variable): A list of :class:`~chainer.Variable`
+            holding input values. Each element ``xs[t]`` holds input value
+            for time ``t``. Its shape is ``(B_t, I)``, where ``B_t`` is
+            mini-batch size for time ``t``, and ``I`` is size of input units.
+            Note that this functions supports variable length sequences.
+            When sequneces has different lengths, sort sequences in descending
+            order by length, and transpose the sorted sequence.
+            :func:`~chainer.functions.transpose_sequence` transpose a list
+            of :func:`~chainer.Variable` holding sequence.
+            So ``xs`` needs to satisfy
+            ``xs[t].shape[0] >= xs[t + 1].shape[0]``.
+        train (bool): If ``True``, this function executes dropout.
+        use_cudnn (bool): If ``True``, this function uses cuDNN if available.
+        activation (str): Activation function name.
+            Please select ``tanh`` or ``relu``.
+
+    Returns:
+        tuple: This functions returns a tuple concaining three elements,
+            ``hy`` and ``ys``.
+            - ``hy`` is an updated hidden states whose shape is same as ``hx``.
+            - ``ys`` is a list of :class:`~chainer.Variable` . Each element
+              ``ys[t]`` holds hidden states of the last layer corresponding
+              to an input ``xs[t]``. Its shape is ``(B_t, N)`` where ``B_t`` is
+              mini-batch size for time ``t``, and ``N`` is size of hidden
+              units. Note that ``B_t`` is the same value as ``xs[t]``.
+
+  .. seealso::
+      :func:`chainer.functions.n_step_rnn_base`
+    """
+    return n_step_rnn_base(n_layers, dropout_ratio, hx, ws, bs, xs, train,
+                           use_cudnn, activation, use_bi_direction=False)
+
+
+def n_step_birnn(n_layers, dropout_ratio, hx, ws, bs, xs, train=True,
+                 use_cudnn=True, activation='tanh'):
+    """Stacked RNN function for sequence inputs.
+
+    This function calculates stacked RNN with sequences. This function gets
+    an initial hidden state :math:`h_0`, an initial cell state :math:`c_0`,
+    an input sequence :math:`x`, weight matrices :math:`W`, and bias vectors
+    :math:`b`.
+    This function calculates hidden states :math:`h_t` and :math:`c_t` for each
+    time :math:`t` from input :math:`x_t`.
+
+    .. math::
+    hf_t &=& f(Wf_0 x_t + Wf_1 hf_{t-1} + bf_0 + bf_1), \\\\
```

Comment:
`Wf` is not valid in equation. Use other representation such as `W^(f)` or `\\vec{W}`